model/graphql/registry.yaml

```
@@ -31,5 +31,5 @@ groups:
         brief: "The GraphQL document being executed."
         type: string
         stability: development
-        note: The value may be sanitized to exclude sensitive information.
+        note: The value SHOULD be sanitized to exclude sensitive information.
```

I agree with the intention, but I don't think it's implementable in the current shape.

some suggestions (please review my English):


```suggestion
        note: If instrumentation can reliably identify and redact sensitive information is SHOULD do it.
```

or 

```suggestion
        note: Application developers should sanitize this attribute in telemetry processing pipeline to exclude sensitive information.
```

(we don't use normative language when talking about app developers - we don't have authority over them)